inference-engine/samples/hello_query_device/README.md

```diff
@@ -48,6 +48,22 @@ Available devices:
 		EXCLUSIVE_ASYNC_REQUESTS : false
 		DLIA_IO_TRANSFORMATIONS_NATIVE : false
 		DLIA_PERF_ESTIMATION : true
+    Device: VPUX
+    Metrics:
+        AVAILABLE_DEVICES : [ 0-2490 ]
+        SUPPORTED_METRICS : [ SUPPORTED_METRICS AVAILABLE_DEVICES FULL_DEVICE_NAME SUPPORTED_CONFIG_KEYS OPTIMIZATION_CAPABILITIES RANGE_FOR_ASYNC_INFER_REQUESTS RANGE_FOR_STREAMS ]
```

Comment:
SUPPORTED_METRICS twice?

Reply:
`SUPPORTED_METRICS` metric includes itself. It might be confusing but looks logically correct from my point of view.

Please see output for other devices in the `README.md`.